models/Result.js

```diff
@@ -333,43 +356,100 @@ module.exports = class Results {
     });
   }
 
-  parse(result, nameNumber) {
-    return {
-      comments: result.comments,
-      opponentName: result.opponentname,
-      performanceDate: moment(result.performdate).format('MMMM D, YYYY'),
-      performanceId: result.performanceid,
-      performanceName: result.name,
-      spotId: result.spotid,
-      winner: nameNumber === result.winnerid
-    };
+  get id() {
+    return this._id;
   }
 
-  parseForAdmin(result) {
-    return {
-      id: result.resultid,
-      firstComments: result.firstcomments,
-      firstName: result.nameone,
-      firstNameNumber: result.firstnamenumber,
-      pending: result.pending,
-      performanceId: result.performanceid,
-      performanceName: result.performancename,
-      secondComments: result.secondcomments,
-      secondName: result.nametwo,
-      secondNameNumber: result.secondnamenumber,
-      spotId: result.spotid,
-      winner: result.winnerid === result.namenumberone ? result.nameone : result.nametwo
-    };
+  get firstComments() {
+    return this._firstComments;
   }
 
-  parseForEval(result) {
+  get firstNameNumber() {
+    return this._firstNameNumber;
+  }
+
+  get needsApproval() {
+    return this._needsApproval;
+  }
+
+  get opponentName() {
+    return this._opponentName;
+  }
+
+  get pending() {
+    return this._pending;
+  }
+
+  get performanceId() {
+    return this._performanceId;
+  }
+
+  get secondComments() {
+    return this._secondComments;
+  }
+
+  get secondNameNumber() {
+    return this._secondNameNumber;
+  }
+
+  get spotId() {
+    return this._spotId;
+  }
+
+  get userOneAlternate() {
+    return this._userOneAlternate;
+  }
+
+  get userTwoAlternate() {
+    return this._userTwoAlternate;
+  }
+
+  // This function is only for results returned by findForUser
+  toJSONForUser(nameNumber) {
     return {
-      firstName: result.nameone,
-      firstNameNumber: result.firstnamenumber,
-      resultId: result.resultid,
-      secondName: result.nametwo,
-      secondNameNumber: result.secondnamenumber,
-      spotId: result.spotid
+      comments: this._firstComments,
+      opponentName: this._opponentName,
+      performanceDate: moment(this._performanceDate).format('MMM D, YYYY'),
+      performanceName: this._performanceName,
+      spotId: this._spotId,
+      winner: this._winnerId === nameNumber
     };
   }
-};
+}
+
+// id, fC, fNN, nA, opponentName, pending, perfDate, perfName, perfId, sC, sNN, spotId, winnerId
+const instanceFromDBRow = ({
```

Comment:
I haven't yet figured out a non-annoying way to do this. Let me know if you find one.